source/clear-linux/guides/network/custom-clear-container.rst

```diff
@@ -0,0 +1,378 @@
+  Docker to work.
+* Basic knowledge of Docker is required.
+
+Build the base Clear Linux container image
+******************************************
+
+#. Log in and get root privileges.
+
+   .. code-block:: console
+
+      $ sudo -s
+
+#. Verify Docker is installed and running.
+
+   .. code-block:: console
+
+      # docker info
+
+   If Docker is installed and running, the expected output will be similar to
+   this:
+
+   .. code-block:: console
+
+      # docker info
+
+      Containers: 0
+       Running: 0
+       Paused: 0
+       Stopped: 0
+      Images: 4
+      Server Version: 17.05.0-ce
+      Storage Driver: overlay
+       Backing Filesystem: extfs
+       Supports d_type: true
+      Logging Driver: json-file
+      Cgroup Driver: cgroupfs
+      Plugins:
+       Volume: local
+       Network: bridge host macvlan null overlay
+      Swarm: inactive
+      Runtimes: runc
+      Default Runtime: runc
+      Init Binary: docker-init
+      containerd version:  (expected: 9048e5e50717ea4497b757314bad98ea3763c145)
+      runc version: N/A (expected: 9c2d8d184e5da67c95d601382adf14862e4f2228)
+      init version: N/A (expected: )
+      Kernel Version: 4.12.7-377.native
+      Operating System: Clear Linux OS for Intel Architecture
+      OSType: linux
+      Architecture: x86_64
+      CPUs: 4
+      Total Memory: 15.62GiB
+      Name: clr-os
+      ID: XQHJ:DYEM:3Q4D:DKLM:JOA4:RUSF:GAFR:DLPA:HOJP:W5FF:ULEE:7HZ3
+      Docker Root Dir: /var/lib/docker
+      Debug Mode (client): false
+      Debug Mode (server): false
+      Registry: https://index.docker.io/v1/
+      Experimental: false
+      Insecure Registries:
+       127.0.0.0/8
+      Live Restore Enabled: false
+
+   If Docker is not installed, perform these steps:
+
+   .. code-block:: console
+
+      # swupd bundle-add containers-basic
+      # systemctl start docker
+
+#. Create the directory structure needed for building the |CL| container.
+
+   .. code-block:: console
+
+      # mkdir -p ./custom-clear-linux-container/base/usr/share/clear/bundles
+      # cd custom-clear-linux-container
+
+   .. note::
+
+      * The directories `customer-clear-linux-container` and `base` are for
+        the purpose of staging and can be named something else, if preferred.
+      * The remaining directories (`/usr/share/clear/bundles`) are mandatory.
+
+#. Create the reference files of the minimum required |CL| bundles (`os-core` and
+   `os-core-update`).  `swupd` determines which bundles to download and install
+   by using the reference filenames. 
+
+   .. code-block:: console
+
+      # touch ./base/usr/share/clear/bundles/os-core
+      # touch ./base/usr/share/clear/bundles/os-core-update
+
+   .. note::
+
+      * `os-core` provides the minimal Linux namespace.
+      * `os-core-update` provides basic suite for running the |CL|
+        for iA Updater
+
+#. Optionally, additional bundles can be included with the base image.   
```

Comment:
The content in this step has the incorrect indentation and does not keep the 78 character line length.